infrastructure/terraform/components/app/module_amplify_branch.tf

```diff
@@ -1,14 +1,14 @@
 module "amplify_branch" {
   source            = "git::https://github.com/NHSDigital/nhs-notify-shared-modules.git//infrastructure/modules/amp_branch?ref=v1.0.0"
-  name              = "main"
+  name              = var.branch_name
```

Comment:
Is production branch not always main? why the change?

Reply:
This allows you to override the production branch for occasions where you want to deploy your custom branch rather than main which is likely when testing a whole new dev environment.